### ISSUE TITLE
[Bug]: error while loading shared libraries: /snap/mattermost-desktop/551/gnome-platform/usr/lib/x86_64-linux-gnu/libplds4.so: unsupported version 0 of Verneed record

### ISSUE THREAD

**Issue by dannf** [snapcrafters/mattermost-desktop#60]
### What happened?

Application fails to start.

### What should have happened?

Application starts.

### Output of `snap info $snap_name`

```shell
name:      mattermost-desktop
summary:   Open source, private cloud Slack-alternative
publisher: Snapcrafters
store-url: https://snapcraft.io/mattermost-desktop
contact:   https://github.com/snapcrafters/mattermost-desktop/issues
license:   unset
description: |
  Mattermost is secure workplace messaging from behind your firewall.
  
    - Discuss topics in private groups, one-to-one or team-wide
    - Easily share and view image files
    - Connect in-house systems with webhooks and Slack-compatible integrations
  
  To use this app, you need a URL for a Mattermost server.
  
  -------
  
  Host your own server: https://about.mattermost.com/download
  Terms of Service: http://about.mattermost.com/terms/
  Contribute to the project: https://github.com/mattermost/desktop
  
  
  This snap is maintained by the Snapcrafters community, and is not necessarily endorsed or
  officially maintained by the upstream developers.
commands:
  - mattermost-desktop
snap-id:      ed0pxJoDHrgmAWHH7baX5nryAHy1UNj0
tracking:     latest/edge
refresh-date: today at 07:50 MDT
channels:
  latest/stable:    5.1.1 2022-07-12 (523) 120MB -
  latest/candidate: 5.1.1 2022-08-05 (533) 120MB -
  latest/beta:      ↑                            
  latest/edge:      5.1.1 2022-10-12 (551) 118MB -
installed:          5.1.1            (551) 118MB -
```

### Output of `snap connections $snap_name`

```shell
Interface               Plug                                       Slot                            Notes
audio-playback          mattermost-desktop:audio-playback          :audio-playback                 -
audio-record            mattermost-desktop:audio-record            -                               -
camera                  mattermost-desktop:camera                  -                               -
content[gnome-42-2204]  mattermost-desktop:gnome-42-2204           gnome-42-2204:gnome-42-2204     -
content[gtk-3-themes]   mattermost-desktop:gtk-3-themes            gtk-common-themes:gtk-3-themes  -
content[icon-themes]    mattermost-desktop:icon-themes             gtk-common-themes:icon-themes   -
content[sound-themes]   mattermost-desktop:sound-themes            gtk-common-themes:sound-themes  -
desktop                 mattermost-desktop:desktop                 :desktop                        -
desktop-legacy          mattermost-desktop:desktop-legacy          :desktop-legacy                 -
gsettings               mattermost-desktop:gsettings               :gsettings                      -
home                    mattermost-desktop:home                    :home                           -
login-session-observe   mattermost-desktop:login-session-observe   -                               -
network                 mattermost-desktop:network                 :network                        -
network-bind            mattermost-desktop:network-bind            :network-bind                   -
opengl                  mattermost-desktop:opengl                  :opengl                         -
pulseaudio              mattermost-desktop:pulseaudio              :pulseaudio                     -
removable-media         mattermost-desktop:removable-media         -                               -
screen-inhibit-control  mattermost-desktop:screen-inhibit-control  :screen-inhibit-control         -
shared-memory           mattermost-desktop:shmem                   :shared-memory                  -
unity7                  mattermost-desktop:unity7                  :unity7                         -
wayland                 mattermost-desktop:wayland                 :wayland                        -
x11                     mattermost-desktop:x11                     :x11                            -
```

### Output of `snap version`

```shell
snap    2.57.2
snapd   2.57.2
series  16
debian  -
kernel  6.0.0-rc7-amd64
```

### Teminal output of app

```shell
$ mattermost-desktop 
/snap/mattermost-desktop/551/opt/Mattermost/mattermost-desktop: error while loading shared libraries: /snap/mattermost-desktop/551/gnome-platform/usr/lib/x86_64-linux-gnu/libplds4.so: unsupported version 0 of Verneed record
$
```

**Comment by jnsgruk** [comment]
Hi!

I think this should be resolved with the latest `5.2.2` release. Could you try refreshing to the `latest/candidate` release and let me know?

```
sudo snap refresh mattermost-desktop --channel latest/candidate
```

Thanks!

**Comment by diddledani** [comment]
This is not an issue with this snap package. It is a known bug in Linux 6.0 as distributed from upstream. The issue should be fixed in the kernel release version 6.0.7, which is out now. If your bistro hasn't patched their 6.0 kernel series yet then you should file a bug with them directly and point them to the following:

- https://lore.kernel.org/r/20221102022111.398283374@linuxfoundation.org
- https://lore.kernel.org/lkml/20221020223616.7571-1-phillip@squashfs.org.uk/
